packages/console/src/hooks/use-position.ts

```diff
@@ -0,0 +1,201 @@
+import { RefObject, useCallback, useEffect, useState } from 'react';
+
+export type VerticalAlignment = 'top' | 'bottom';
+
+export type HorizontalAlignment = 'start' | 'center' | 'end';
```

Comment:
nit: would `left | center | right` be better?

Reply:
I think `start` and `end` would be more accurate.
<img width="143" alt="image" src="https://user-images.githubusercontent.com/10806653/168204451-7978e5af-3ac3-41cf-b4d1-25a724cd7888.png">